spec/system/page_objects/pages/user.rb

```diff
@@ -24,9 +24,16 @@ def has_warning_messages_path?(user)
         page.has_current_path?("/u/#{user.username}/messages/warnings")
       end
 
+      def staff_info_section
+        begin
+          page.find(".staff-counters")
+        rescue Capybara::ElementNotFound
```

Comment:
I'm curious what is the reason for rescuing from an `ElementNotFound` error?

Reply:
With the testing of the negative path being handled by the `has_no_staff_info_flagged_posts_counter?` method, all of this is no longer necessary, so I've removed this.